.stylelintrc

```diff
@@ -1,11 +1,15 @@
 {
-  "extends": "stylelint-config-recommended",
+  "extends": [
+    "stylelint-config-recommended",
+    "stylelint-config-sass-guidelines",
+    "stylelint-config-css-modules",
+    "stylelint-config-concentric-order"
+  ],
   "rules": {
-    "selector-pseudo-class-no-unknown": [
-      true,
-      {
-        "ignorePseudoClasses": ["global"]
-      }
-    ]
+    "declaration-property-value-blacklist": null,
+    "max-nesting-depth": null,
```

Comment:
This rule is Sass specific, so disabled to prevent duplicate warnings for nesting depth.